rotkehlchen/accounting/cost_basis/base.py

```diff
@@ -303,9 +300,13 @@ def reduce_asset_amount(self, asset: Asset, amount: FVal, timestamp: Timestamp)
         if remaining_amount_from_last_buy != FVal('-1'):
             asset_events.acquisitions[0].remaining_amount = remaining_amount_from_last_buy
         elif remaining_amount != ZERO:
-            log.critical(
-                f'No documented buy found for {asset} before '
-                f'{self.timestamp_to_date(timestamp)}',
+            self.missing_acquisitions.append(
+                MissingAcquisition(
+                    asset=asset,
+                    time=timestamp,
+                    found_amount=ZERO,
+                    missing_amount=amount,
```

Comment:
This is wrong here. As you can see from the code this happens if we come here with non-zero remaining amount. So the missing amount is that. Plus the found amount is the difference.

```suggestion
                    found_amount=amount - remaining_amount,
                    missing_amount=remaining_amount,
```